src/main/java/org/gridsuite/shortcircuit/server/repositories/ShortCircuitAnalysisResultRepository.java

```diff
@@ -248,20 +251,70 @@ public Optional<ShortCircuitAnalysisResultEntity> findResultsWithLimitViolations
     }
 
     @Transactional(readOnly = true)
-    public Page<FaultResultEntity> findFaultResultsPage(ShortCircuitAnalysisResultEntity result, List<ResourceFilter> resourceFilters, Pageable pageable, FaultResultsMode mode) {
+    public Page<FaultResultEntity> findFaultResultsPage(ShortCircuitAnalysisResultEntity result,
+                                                        List<ResourceFilter> resourceFilters,
+                                                        Pageable pageable,
+                                                        FaultResultsMode mode) {
         Objects.requireNonNull(result);
-        Specification<FaultResultEntity> specification = FaultResultSpecificationBuilder.buildSpecification(result.getResultUuid(), resourceFilters);
+
+        Optional<Sort.Order> childrenSort = extractChildrenSort(pageable);
+
+        Pageable modifiedPageable = addDefaultSort(filterOutChildrenSort(pageable, childrenSort),
+                DEFAULT_FAULT_RESULT_SORT_COLUMN);
+        Specification<FaultResultEntity> specification = faultSpecBuilder.buildSpecification(result.getResultUuid(), resourceFilters);
```

Comment:
You set back the other one to static, this one can be static too

Reply:
Ok done.